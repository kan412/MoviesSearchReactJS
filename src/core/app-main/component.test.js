import React from 'react';
import { shallow } from 'enzyme';
import AppMain from './component';

describe('<AppMain />', () => {
  it('render without breaking', () => {
    const component = shallow(<AppMain />);
    expect(component).toMatchSnapshot();
  });
});
