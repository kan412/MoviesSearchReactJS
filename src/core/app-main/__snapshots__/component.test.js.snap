// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppMain /> render without breaking 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/film/:id"
      />
      <Route
        component={[Function]}
        path="/404"
      />
      <Redirect
        from="*"
        to="/404"
      />
    </Switch>
  </BrowserRouter>
</Provider>
`;
